server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -655,9 +654,64 @@ public Iterator<Setting<?>> settings() {
         Property.Final
     );
 
+    /**
```

Comment:
This change is just a result of reordering settings in such a way that static initialization happens in the right order.